frontend/src/pages/Users.tsx

```diff
@@ -1,143 +1,171 @@
+      .then((response) => {
+        this.setState({
+          users: response.Users,
+          newUser: "",
+          newUserFieldEnabled: true,
         })
-    }
-
-    updateNewUserValue(evt: React.ChangeEvent<HTMLInputElement>) {
+      })
+      .catch((err) => {
+        // Need to indicate error...
         this.setState({
-            newUser: evt.target.value,
-            newUserButtonEnabled: evt.target.value !== "",
+          newUserFieldEnabled: true,
+          newUserButtonEnabled: true, // Not that helpful but probably less confusing?
         })
-    }
-
-    renderUsersTable() {
-        if (this.state.loadingUsers) {
-            return <Loading />
-        }
-        return (
-            <table className="table table-responsive-sm">
-                <thead>
-                    <tr>
-                        <th scope="col">Email Address</th>
-                        <th scope="col">Created By</th>
-                        <th scope="col">Actions</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {this.state.users.map(user =>
-                        <tr key={user.id}>
-                            <th scope="row">{user.email}</th>
-                            <td>{user.updatedBy}</td>
-                            <td><Button variant="secondary" onClick={evt => this.removeUserClick(user.id)}>Delete</Button></td>
-                        </tr>
-                    )}
-                    <tr key="newUser">
-                        <th scope="row">
-                            <input type="text" className="form-control" id="newUser" placeholder="Enter new user's Google email address" value={this.state.newUser} onChange={evt => this.updateNewUserValue(evt)} disabled={!this.state.newUserFieldEnabled} onKeyUp={(evt) => evt.key === "Enter" ? this.newUserClick() : ""}/>
-                        </th>
-                        <td></td>
-                        <td><Button variant="secondary" onClick={() => this.newUserClick()} disabled={!this.state.newUserButtonEnabled}>Create</Button></td>
-                    </tr>
-                </tbody>
-            </table>
-        )
-    }
+      })
+  }
 
+  updateNewUserValue(evt: React.ChangeEvent<HTMLInputElement>) {
+    this.setState({
+      newUser: evt.target.value,
+      newUserButtonEnabled: evt.target.value !== "",
+    })
+  }
 
-    render() {
-        return (
-            <div>
-                <div className="card" style={{marginBottom: "1rem", marginTop: "1rem"}}>
-                    <div className="card-body">
-                    <h2 className="card-title">Users</h2>
-                    {this.renderUsersTable()}
-                    </div>
-                </div>
-            </div>
-        )
+  renderUsersTable() {
+    if (this.state.loadingUsers) {
+      return <Loading />
     }
-}
\ No newline at end of file
+    return (
+      <Table className="table table-responsive-sm">
```

Comment:
I think you can replace the `className` with other attributes - https://react-bootstrap.github.io/components/table/

Reply:
FYI: it's good to resolve these after the fix has been pushed.

Reply:
Looks like you didn't do this yet, were you planning on doing so?

Reply:
https://github.com/anthonywittig/mLock/commit/8ce150ae4e97075209bb9b090d5ca201a5e48314